Library/ViewModels/CommentsViewModelTests.swift

```diff
@@ -111,365 +113,371 @@ internal final class CommentsViewModelTests: TestCase {
 
       self.isCommentComposerHidden.assertValue(true)
     }
+  }
 
-    func testGoToCommentReplies_CommentHasReplies_GoToEmits() {
-      self.goToCommentRepliesComment.assertDidNotEmitValue()
-      self.goToCommentRepliesProject.assertDidNotEmitValue()
+  func testGoToCommentReplies_CommentHasReplies_GoToEmits() {
+    self.goToCommentRepliesComment.assertDidNotEmitValue()
+    self.goToCommentRepliesProject.assertDidNotEmitValue()
 
-      let project = Project.template
-      let comment = Comment.template
-        |> \.replyCount .~ 1
+    let project = Project.template
```

Comment:
If you are not changing the template, I would just delete the constant and supply an argument of `.template` wherever it's needed.

Reply:
Check all instances of this below as well.

Reply:
Oh I didn't noticed it, I only copied out all the nested test. I'll remove the instances.